pkgs/development/python-modules/jinja2/2.nix

```diff
@@ -23,8 +23,14 @@ buildPythonPackage rec {
   # See https://github.com/pallets/jinja/issues/1158
   doCheck = !stdenv.is32bit || isPy3k;
 
-  checkPhase = ''
-    pytest -v tests -W ignore::ResourceWarning -W ignore::DeprecationWarning
+  checkPhase = let
+    pytestArgs = lib.optionals isPy3k [
```

Comment:
Let's not add python3 logic into a python2 only package.

Reply:
Not sure what you mean? Seems py2/3 compatible, there's another isPy3k conditional before this.

Reply:
I duplicated this version before upgrading the original package to a python2-incompatbile one. Python 3 uses the `default.nix`, while Python2 uses `2.nix`. All Python3 logic can be dropped from `2.nix`.

Reply:
Oh I see, thanks